account_financial_report_qweb/wizard/journal_report_wizard.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class JournalReportWizard(models.TransientModel):
+
+    _name = 'journal.report.wizard'
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        default=lambda self: self.env.user.company_id,
+        string='Company'
+    )
+    date_range_id = fields.Many2one(
```

Comment:
Can you add a filter on the company ?

Reply:
ondelete='cascade'

Reply:
If you meant to add a filter in the excel report, I just did it.